app/services/fastboot.js

```diff
@@ -75,12 +73,6 @@ const FastBootService = Ember.Service.extend({
   },
 
   host: computed(function() {
```

Comment:
That refers to `fastboot.request.host`. This is `fastboot.host` if I'm not mistaken.